src/scripts/underground/UndergroundUpgrade.ts

```diff
@@ -20,9 +20,13 @@ namespace UndergroundUpgrade {
     export enum Upgrades {
         'Energy_Max',
         'Items_Max',
+        'Items_Min',
```

Comment:
By moving this, I _think_ that you will also need to update the players purchased upgrades in their save data in `Update.ts`

Reply:
I'll look into this too, would my new upgrades need to be added to Update.ts too? 
I'll check.

Reply:
Hmm, I tried importing one of my old saves, I didn't appear to lose any my purchased upgrades.